src/core/sync/mutex.d

```diff
@@ -55,62 +57,92 @@ class Mutex :
     /**
      * Initializes a mutex object.
      *
-     * Throws:
-     *  SyncError on error.
      */
-    this() nothrow @trusted
+    this() @trusted nothrow @nogc
+    {
+        this(true);
+    }
+
+    /// ditto
+    this() shared @trusted nothrow @nogc
```

Comment:
@ZombineDev 
Unfortunately this broke sub-classes, see [Issue 17130 – [Reg 2.074] ambiguous implicit super call when inheriting core.sync.mutex.Mutex](https://issues.dlang.org/show_bug.cgi?id=17130). Would be nicer to fix this in dmd though.